### PR TITLE
Adding cask kinesis-smartset v2.1.5

### PR DESCRIPTION
**Pull request by outofband**
After making all changes to the cask:

- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` reports no offenses.
- [X] The commit message includes the cask’s name and version.
- [X] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [X] Named the cask according to the [token reference].
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.
- [X] Checked there are no [open pull requests] for the same cask.
- [X] Checked the cask was not [already refused].
- [X] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/kinesis-smartset.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  version '2.1.5'
+  sha256 'b023dfb1521dfc4b58d1fa8d3669410d44fc375b8795cc0aea6eec18da4a2f4a'
+
+  url "https://gaming.kinesis-ergo.com/wp-content/uploads/2019/10/SmartSet-App-for-Freestyle-Edge-RGB-for-macOS-Version-#{version}-64-Bit.zip"
+  name 'Kinesis SmartSet'
+  homepage 'https://gaming.kinesis-ergo.com/fs-edge-rgb-support/#smartset'
+
+  app "SmartSet App for Freestyle Edge RGB for macOS (Version #{version}) (64 Bit).app"
+
+  caveats 'The SmartSet App is currently unsigned, so you will need to approve it in System Preferences>Security.'
```
Comment (vitorgalvao) [inline Casks/kinesis-smartset.rb]: Remove the caveat.

File: Casks/kinesis-smartset.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+cask 'kinesis-smartset' do
```
Comment (vitorgalvao) [inline Casks/kinesis-smartset.rb]: Needs to be a tokenization of `SmartSet App for Freestyle Edge RGB`. Yes, it’s an awful name; it’s also the one they chose.

File: Casks/kinesis-smartset.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  version '2.1.5'
+  sha256 'b023dfb1521dfc4b58d1fa8d3669410d44fc375b8795cc0aea6eec18da4a2f4a'
+
+  url "https://gaming.kinesis-ergo.com/wp-content/uploads/2019/10/SmartSet-App-for-Freestyle-Edge-RGB-for-macOS-Version-#{version}-64-Bit.zip"
```
Comment (vitorgalvao) [inline Casks/kinesis-smartset.rb]: Please add an [`appcast`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md): `https://gaming.kinesis-ergo.com/fs-edge-rgb-support/#smartset`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by core-code** [comment]
is there any rule how long we should keep waiting for things being stuck at 'awaiting user reply' before closing them?

**Comment by vitorgalvao** [comment]
> is there any rule how long we should keep waiting for things being stuck at 'awaiting user reply' before closing them?

No rule. There was a bot (I think it’s timing was 21 days), but it seems to no longer be active. Do whatever feels right, depending on the conversation and previous replies.

Almost three months is too long.

**Pull request closed without merging**